### PR TITLE
Added remaining linux/wireless.h struct and constants

### PR DESCRIPTION
**Pull request by Brijeshkrishna**
Added remaining wireless struct and constants to Linux.

### CODE REVIEW

**Comment by Brijeshkrishna** [comment]
@JohnTitor any changes

**Comment by Brijeshkrishna** [comment]
@JohnTitor ?

File: src/unix/linux_like/linux/mod.rs [hunk 1]
```diff
@@ -2965,6 +3203,16 @@ pub const SIOCIWLASTPRIV: ::c_ulong = 0x8BFF;
...
 
+pub fn IW_IOCTL_IDX(cmd: ::c_ulong) -> ::c_ulong {
+    cmd - SIOCIWFIRST
+}
+pub fn IW_IS_SET(cmd: ::c_ulong) -> ::c_ulong {
+    !(cmd & 0x1)
+}
+pub fn IW_IS_GET(cmd: ::c_ulong) -> ::c_ulong {
+    cmd & 0x1
+}
```
Comment (JohnTitor) [inline src/unix/linux_like/linux/mod.rs:3214]: This crate should be a thin wrapper and I'd like to remove them.

File: src/unix/linux_like/linux/mod.rs [hunk 2]
```diff
@@ -3165,6 +3416,37 @@ pub const IW_ENC_CAPA_CIPHER_TKIP: ::c_ulong = 0x00000004;
...
+macro_rules! IW_EVENT_CAPA_SET_KERNEL {
+    // It is not in libc-test/semver/linux.txt
+    ($event_capa:expr) => {{
+        {
+            event_capa[0] |= IW_EVENT_CAPA_K_0;
+            event_capa[1] |= IW_EVENT_CAPA_K_1;
+        }
+    }};
+}
```
Comment (JohnTitor) [inline src/unix/linux_like/linux/mod.rs:3448]: Same here

File: src/unix/linux_like/linux/mod.rs [hunk 3]
```diff
@@ -3173,9 +3455,23 @@ pub const IW_PMKID_LEN: usize = 16;
...
+pub const IW_EV_QUAL_LEN: usize = IW_EV_LCP_LEN + ::mem::size_of::<iw_quality>();
+
 pub const IW_EV_LCP_PK_LEN: usize = 4;
 
 pub const IW_EV_CHAR_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::IFNAMSIZ;
+pub const IW_EV_UINT_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<u32>();
+pub const IW_EV_FREQ_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<iw_freq>();
+pub const IW_EV_PARAM_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<iw_param>();
+pub const IW_EV_ADDR_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<::sockaddr>();
+pub const IW_EV_QUAL_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<iw_quality>();
```
Comment (JohnTitor) [inline src/unix/linux_like/linux/mod.rs:3474]: Same here

Comment (Brijeshkrishna) [reply src/unix/linux_like/linux/mod.rs:3474]: Can I set a fixed value? Instead of calculating that.

Comment (JohnTitor) [reply src/unix/linux_like/linux/mod.rs:3474]: If it works, yeah.

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thanks! @bors r+

**Comment by JohnTitor** [comment]
Could you rebase onto main instead of merging? The latest main will trigger full CI.

**Comment by Brijeshkrishna** [comment]
@JohnTitor , I think auto merge is not working

**Comment by JohnTitor** [comment]
Sorry! It should work now.

**Pull request merged**